sn/config.yaml

```diff
@@ -39,18 +39,21 @@ storage:
   shard_num: 1
   shard:
     0:
+      resync_metabase: true
       writecache:
         enabled: false
       metabase:
         path: /data/metabase.db
         perm: 0770
       blobstor:
         - type: blobovnicza
-          path: /data/blob
+          path: /data/blob/blobovnicza
           perm: 0660
           opened_cache_capacity: 32
-          depth: 1
+          depth: 2
```

Comment:
Why do we need this change?

Reply:
Because `neofs-node v.0.34.0` requires 2 components in blobstor, otherwise it raises an error:
```
 blobstor section must have 2 components, got: 1
```
And configuration with 2 components and depth 1 leads to the following error:
```
  error   engine/control.go:81    shard initialization failure, skipping  {"id": "MQhwV2e4rzPkFu3S8XxsSs", "error": "could not initialize *blobstor.BlobStor: failure on blobovnicza initialization stage: could not open blobovnicza 0/0: open /data/blob/blobovnicza/0/0: is a directory"}
```

Reply:
The error reveals a bug in blobstor or your previous configuration had another depth|width. Changing the config value in source code isn't a pill.

Reply:
Ok, I thought the error appeared because of increasing a number of components. I will open an issue.

Reply:
Let's mention in docs about volume cleaning before update and keep depth:1 width:1 parameters.